script.js

```diff
@@ -125,16 +131,18 @@ operators.forEach(operator => {
     operator.addEventListener("click", () => {
         storeNonEmptyCurrentDisplay();
         const clicked = operator.textContent;
+        lastOperator = clicked;
```

Comment:
Stored first, check if needed, because lastOperator will be stored in = if condition fulfilled.